apps/OpenSpace/ext/launcher/src/sgctedit/windowcontrol.cpp

```diff
@@ -297,35 +334,52 @@ void WindowControl::createWidgets(const QColor& windowColor) {
             "images available to other real-time applications on the same machine for "
             "further processing"
         );
-        layout->addWidget(_spoutOutput);
+        layout->addWidget(_spoutOutput, 6, 0, 1, 4);
     }
+
+    //
+    // Render 2D & 3D
     {
-        QFrame* projectionGroup = new QFrame;
-        projectionGroup->setVisible(true);
-        projectionGroup->setFrameStyle(QFrame::StyledPanel | QFrame::Plain);
+        _render2D = new QCheckBox("Render Overlays");
```

Comment:
I found it a bit difficult to understand that the "overlays" in this case also include the UI. Can we maybe make this clearer in some way? Perhaps something like this is enough? It feel like that's one of the settings that many users would be looking for so we should make it easy to find
```suggestion
        _render2D = new QCheckBox("Render overlays (including GUI)");
```

Also, why do we use title case instead of sentence case on all the inputs?  DO we want to keep this? Then my suggestion above needs to be revised accordingly

Reply:
Went with the second option.  Don't have many thought about titlecase or not;  it was like that before and didn't feel it needed changing